src/test/test-cgroup-util.c

```diff
@@ -426,4 +426,13 @@ TEST(cg_get_keyed_attribute) {
         }
 }
 
+TEST(bfq_weight_conversion) {
+        assert_se(BFQ_WEIGHT(1) == 1);
+        assert_se(BFQ_WEIGHT(50) == 50);
+        assert_se(BFQ_WEIGHT(100) == 100);
+        assert_se(BFQ_WEIGHT(500) == 136);
+        assert_se(BFQ_WEIGHT(5000) == 545);
+        assert_se(BFQ_WEIGHT(10000) == 1000);
```

Comment:
No, please drop this commit. It replaces a **static** test by something that actually emits code, even if not much. The static test is done by the compiler, immediately and without any further cost. And if the tests are inline in cgroup.c, there is no need to export the macros.

Reply:
I was aware of the tests having been evaluated at compile time.
There's no free lunch, everyone who just compiles will need to exert this work, whereas with the extraction, it's only run when the tests are actually wanted.

Perhaps the tests could be made static in `test-cgroup-util.c`?

Reply:
Well, I hope that anyone who compiles this at least runs the unit tests… It's a basic check that you should always do. But even if not, I just don't think this matters: static asserts are essentially free compared to the costs of loading the code from disk and compiling and linking.